packages/qwik-city/utils/test-suite.ts

```diff
@@ -1,21 +1,23 @@
 /* eslint-disable no-empty-pattern */
-import { assert, test, beforeAll, type TestAPI } from 'vitest';
+import { tmpdir } from 'node:os';
 import { join } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { assert, beforeAll, test, type TestAPI } from 'vitest';
+import { build } from '../buildtime/build';
+import { createBuildContext } from '../buildtime/context';
 import type {
   BuildContext,
   BuildLayout,
   BuildRoute,
   MarkdownAttributes,
   PluginOptions,
 } from '../buildtime/types';
-import { createBuildContext } from '../buildtime/context';
-import { tmpdir } from 'node:os';
 import { normalizePath } from './fs';
-import { build } from '../buildtime/build';
-import { fileURLToPath } from 'node:url';
 
 export { assert };
 
+const __dirname = fileURLToPath(new URL('.', import.meta.url));
```

Comment:
we should drop support of node 18
in 20/22 we can use import.meta.dirname

Reply:
let's add a different issue about that where we could discuss this
I just moved this line to the top for now